snuba/util.py

```diff
@@ -255,7 +255,8 @@ def create_metrics(prefix: str, tags: Optional[Tags] = None) -> MetricsBackend:
     from snuba.utils.metrics.backends.datadog import DatadogMetricsBackend
 
     return DatadogMetricsBackend(
-        DogStatsd(
+        partial(
```

Comment:
I have honestly no idea why mypy is OK with this but was complaining about what I was trying to do earlier with the stream processing strategies.

Reply:
What if the difference was that you are not actually passing any parameter to the partial; when you invoke it? It could easily be just a function that is not partially applied.
Though mypy is fine with this which is weird.
https://github.com/getsentry/snuba/blob/master/snuba/web/query.py#L125-L137